### PR TITLE
Add purge.bash

### PR DESCRIPTION
**Pull request by ingkebil**
Added a simple script that will make moneypenny work. It searches for all `qc_sampleInfo.yaml` files and checks if a case is archived. If so, it asks moneypenny to purge the fastq files.

### CODE REVIEW

File: scripts/purge.bash [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+INDIR=$1
+
+for QCFILE in $(find $INDIR -name '*qc_sampleInfo.yaml' -exec grep -H Archived {} \; | cut -d: -f1); do
+   IFS=/ QCFILE_PART=( ${QCFILE} )
```
Comment (ingkebil) [inline scripts/purge.bash]: OK, this is how you create an array with delimiter `/`

File: scripts/purge.bash [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+INDIR=${1?'Please provide full path to MIP_ANALYSIS'}
+CONFIG=${2?'Please provide full path to MP config'}
+
+for QCFILE in $(find $INDIR -name '*qc_sampleInfo.yaml' -exec grep -H Archived {} \; | cut -d: -f1); do
+   IFS=/ QCFILE_PART=( ${QCFILE} )
+   CUST=${QCFILE_PART[6]}
+   CASE=${QCFILE_PART[7]}
+
+   echo "moneypenny -c $CONFIG flow add purge ${CUST} ${CASE}"
```
Comment (ingkebil) [inline scripts/purge.bash]: I guess it doesn't need a config when started with the right user, right? `moneypenny` is an alias with the right config preselected?

**Comment by ingkebil** [comment]
Sorry about the merge mess. Had set the wrong upstream branch :/

File: moneypenny/flow/mutations/purge.py [hunk 3]
```diff
@@ -20,11 +20,23 @@ def _find_same_file(filename, start_dir):
     """
     command = 'find {start_dir} -samefile {filename}'.\
               format(start_dir=start_dir, filename=filename)
-    target_files = [ file for file in \
-                     subprocess.check_output(command.split(' ')).split('\n') if len(file) ]
+    try: 
+        target_files = [ file for file in \
```
Comment (robinandeer) [inline moneypenny/flow/mutations/purge.py]: you don't need `\` 😉

**Comment by robinandeer** [comment]
As far as I can tell it looks good!

**Pull request merged**